Dockerfile

```diff
@@ -1,35 +1,51 @@
+# ---- Base Node ----
 FROM node:18.13.0-bullseye-slim as base
 
-RUN set -ex; \
-    apt update -y; \
-    apt install ca-certificates -y; \
-    update-ca-certificates
-
-FROM base as build
-
-WORKDIR /app
-
-RUN apt install make gcc g++ python3 git openssl -y;
+RUN mkdir -p /home/node/app
+RUN chown -R node:node /home/node && chmod -R 770 /home/node
+WORKDIR /home/node/app
+
+# ---- Dependencies ----
+FROM base AS dependencies
+
+COPY --chown=node:node ./package.json ./package.json prisma ./
+COPY --chown=node:node ./package-lock.json ./package-lock.json
+RUN apt-get update -y && \
```

Comment:
Move installing OS dependencies step before copying package.json and prisma, otherwise it will be run every time Node.js dependencies or migrations change

Reply:
Thanks, missed it when refactoring.